### ISSUE TITLE
Turn off low light compensation?

### ISSUE THREAD

**Issue by hzcyf** [realsenseai/realsense-ros#2990]
![image](https://github.com/IntelRealSense/realsense-ros/assets/28844563/090747bc-63c5-4069-ae24-605e307279a3)
The low light compensation cannot be controlled because there is no API to do so, even though it is listed as supported in the data sheet. Why is this the case and how can it be turned off or on?

**Comment by MartyG-RealSense** [comment]
Hi @hzcyf  I carried out research of Low Light Compensation for you.  It was introduced in June 2018 in the early SDK version 2.13.0 and is described as a **metadata** parameter.  This means it is not a settable option and instead returns a status about whether Low Light Compensation is off (0) or on (1).  

https://intelrealsense.github.io/librealsense/doxygen/rs__frame_8h.html#a91f19a01f5bf2abadc30959a8d3109c9ac726c1023135dddbfe7a30cb46f7932b

![image](https://github.com/IntelRealSense/realsense-ros/assets/41145062/58f48112-2e82-4cc4-bc89-66572331de90)

I checked the SDK documentation and the early versions of the RealSense Viewer tool and there was never a configurable option for Low Light Compensation, which is an RGB color characteristic.  This indicates that Low Light Compensation is simply a metadata parameter that provides confirmation of whether compensation is currently active.  There is no information about what determines whether it is set to on or off.
